http/swagger.yml

```diff
@@ -4805,7 +4805,7 @@ paths:
             schema:
               $ref: "#/components/schemas/LabelMapping"
       responses:
-        '200':
+        '201':
```

Comment:
This was updated to reflect what the code is actually returning:

https://github.com/influxdata/influxdb/blob/master/http/task_service.go#L149

Which, when successful, returns:

https://github.com/influxdata/influxdb/blob/master/http/label_service.go#L406

Which returns a `201`